client/styles/components/_form-container.scss

```diff
@@ -19,6 +19,10 @@
 	flex-direction: column;
 	justify-content: center;
 	align-items: center;
+	width:95%;
```

Comment:
rather than doing a percent width here, i think it makes sense to add a fixed padding or margin.